### PR TITLE
Update config.py

### PR DESCRIPTION
**Pull request by emlynjdavies**
fixes issue #55 

for silcams with adjustable path lengths, this is updated via serial communication using the PathLength class, and vary rarely tested because this hardware needs to be connected. This is done for the first time within  pysilcam/config.py (because the path length is an important configuration parameter for correctly scaling concentrations).

There was a bug here, due to circular imports, where the PathLength class was in the pysilcam.oilgas module - but pysilcam.oilgas also imported pysilcam-config. This PR moves the PathLength class code from pysilcam.oilgas into pysilcam.pathadjust

### CODE REVIEW

**Review by tankefugl — DISMISSED** [review]
No linting violations have been found in this PR.

File: pysilcam/pathadjust.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import serial
+import time
+import struct
+import logging
+
+logger = logging.getLogger(__name__)
+
+class PathLength():
```
Comment (tankefugl) [inline pysilcam/pathadjust.py:8]: E302: expected 2 blank lines, found 1

File: pysilcam/pathadjust.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        packets.append(packitem6)
+        packets.insert(0, packitem1)
+        sendstring = bytearray(bytearray(packets)).decode('latin-1')
+        return sendstring
+
+    def motoronoff(self, ser, state):
+        sendstring = self.makepacket('X', 2, state)
+        self.ser.write(bytes(sendstring, 'latin-1'))
+        time.sleep(0.1)
+        readout1 = self.ser.read(1000)
```
Comment (tankefugl) [inline pysilcam/pathadjust.py:66]: F841: local variable 'readout1' is assigned to but never used

File: pysilcam/pathadjust.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            logger.info('motor is ON')
+        else:
+            logger.info('Something fishy is going on')
+
+    def move(self, ser, newpos):
+        newpos = self.convert_pos(int(newpos))
+        sendstring = self.makepacket('S', 5, newpos)
+        self.ser.write(bytes(sendstring, 'latin-1'))
+        time.sleep(0.1)
+        readout1 = self.ser.read(1000)
```
Comment (tankefugl) [inline pysilcam/pathadjust.py:85]: F841: local variable 'readout1' is assigned to but never used

File: pysilcam/pathadjust.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        self.move(self.ser, 4)
+
+    def gap_to_mm(self, mm):
+        val = 108.5 - mm
+        self.move(self.ser, val)
+
+    def finish(self):
+        self.motoronoff(self.ser, 0)
+        self.ser.close()
+        logger.info('actuator port closed!')
```
Comment (tankefugl) [inline pysilcam/pathadjust.py:119]: W292: no newline at end of file

**Review by tankefugl — CHANGES_REQUESTED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Review by tankefugl — APPROVED** [review]
No linting violations have been found in this PR.

**Comment by emlynjdavies** [comment]
for silcams with adjustable path lengths, this is updated via serial communication using the PathLength class, and vary rarely tested because this hardware needs to be connected. This is done for the first time within  pysilcam/config.py (because the path length is an important configuration parameter for correctly scaling concentrations).

There was a bug here, due to circular imports, where the PathLength class was in the pysilcam.oilgas module - but pysilcam.oilgas also imported pysilcam-config. This PR moves the PathLength class code from pysilcam.oilgas into pysilcam.pathadjust

**Review by bjarnekvae — APPROVED** [review]

**Pull request merged**